src/page/Login/index.jsx

```diff
@@ -35,41 +35,40 @@ export default function Login() {
 
   return (
     <LoginUser>
-      <AiFillQuestionCircle className="login__icon-help"/>
-      <h1 className="login__mainTitle">Feeder</h1>
-      <h2 className="login__subTitle">Project</h2>
+      <AiFillQuestionCircle className='login__icon-help' />
+      <div className='login__container-title'>
+        <h1 className='login__mainTitle'>Feeder</h1>
+        <h2 className='login__subTitle'>Project</h2>
+      </div>
```

Comment:
Se esta usando Styled(CSS in JS) o que vc acha de mantermos o css todo dentro do styled em vez de criar classNames